### PR TITLE
Enables NoCalibratorLinearSvmTest

### PR DESCRIPTION
**Pull request by Anipik**
Related to #78

Test is enabled and corresponding Zbaseline files are added

cc @shauheen @Ivanidzo4ka @eerhardt @danmosemsft @codemzs

### CODE REVIEW

File: test/Microsoft.ML.Predictor.Tests/TestPredictors.cs [hunk 1]
```diff
@@ -1058,21 +1058,26 @@ public void CalibratorPerceptronTest()
...
+        {
+            var datasets = GetDatasetsForCalibratorTest();
+            RunAllTests( new[] { TestLearners.linearSVM }, datasets, new string[] { "cali={}" }, "nocalibration");
+            Done();
+        }
+
+        /// <summary>
+        ///A test for PAV calibrators
         ///</summary>
         [Fact(Skip = "Need CoreTLC specific baseline update")]
```
Comment (eerhardt) [inline test/Microsoft.ML.Predictor.Tests/TestPredictors.cs]: Is there a reason this one is still skipped?

**Comment by Anipik** [comment]
I am enabling them in another PR (already open)

File: test/Microsoft.ML.Predictor.Tests/TestPredictors.cs (hunk 1 above)
Comment (Anipik) [reply test/Microsoft.ML.Predictor.Tests/TestPredictors.cs]: They are getting enabled in #233

Comment (TomFinley) [reply test/Microsoft.ML.Predictor.Tests/TestPredictors.cs]: Actually I'm not sure I understand the structure here... I just came from #233, looked pretty straightforward, but why do we have a separate PR for this? Did we mean to close this PR?

---
In reply to: [190633420](https://github.com/dotnet/machinelearning/pull/232#discussion_r190633420) [](ancestors = 190633420)

Comment (Anipik) [reply test/Microsoft.ML.Predictor.Tests/TestPredictors.cs]: @TomFinley I tried to enable one test at a time. Some of the existing tests are testing multiple models so i split them in the PR eg #229 #232 
I should have waited to create a PR for these new tests eg. #230 and #231 depends upon #229 

#227 #228 #231 #233 needs to merged. #230  would need to be rebased after that.

Comment (TomFinley) [reply test/Microsoft.ML.Predictor.Tests/TestPredictors.cs]: OK. Sorry to hear this. In that case please cancel them all. Rephrase them as *independent* PRs.

So let's think about what is actually happening here. These tests are, what, about three lines long each, and run independently of each other. The idea that the PRs for this somehow "need" to be nested does not make sense to me.

Let's go with this: goal should be to phrase PRs that are easy to review. Not too large. But large enough so I don't have to look a 7 separate PRs, each of which has perhaps two lines different between them and are all touching the same lines, just to figure out what is going on.

Comment (Anipik) [reply test/Microsoft.ML.Predictor.Tests/TestPredictors.cs]: > OK. Sorry to hear this. In that case please cancel them all. Rephrase them as independent PRs.

#227 #228 #231 #233 are already independent PRs

**Pull request closed without merging**